includes/admin/class-wc-payments-admin.php

```diff
@@ -672,8 +693,14 @@ public function enqueue_payments_scripts() {
 						'formattedRefundAmount' => wp_strip_all_tags( wc_price( $refund_amount, [ 'currency' => $order->get_currency() ] ) ),
 						'refundedAmount'        => $order->get_total_refunded(),
 						'canRefund'             => $this->wcpay_gateway->can_refund_order( $order ),
+						'chargeId'              => $this->order_service->get_charge_id_for_order( $order ),
```

Comment:
This shouldn't risk a fatal error, since it is just a wrapped call to `order->get_meta()`.